specification/types.md

```diff
@@ -93,3 +93,13 @@ A structure containing the following fields:
 A structure which supports definition of arbitrary properties, with keys of type `string`, and values of type `boolean`, `string`, or `number`.
 
 This structure is populated by a provider for use by an [Application Author](./glossary.md#application-author) (via the [Evaluation API](./glossary.md#evaluation-api)) or an [Application Integrator](./glossary.md#application-integrator) (via [hooks](./sections/04-hooks.md)).
+
+### Provider Status
+
+An enumeration of possible provider states.
+
+| Status    | Explanation                                                                     |
+| --------- | ------------------------------------------------------------------------------- |
+| NOT_READY | The provider has not been initialized.                                          |
+| READY     | The provider has been initialized, and is able to reliably resolve flag values. |
+| ERROR     | The provider is initialed but is not able to reliably resolve flag values.      |
```

Comment:
I think we have a typo here with "initialed".

Reply:
```suggestion
| ERROR     | The provider is initialized but is not able to reliably resolve flag values.      |
```